aplib.net-demo/Assets/Scripts/Entities/Enemy/MeleeEnemy.cs

```diff
@@ -70,6 +75,7 @@ private void StartSwing()
     {
         _isSwinging = true;
         transform.localScale *= _sizeIncrease;
+        _animator.Play("Charge-up");
```

Comment:
Is it possible to just let the animator depend on variables present within the enemy? Adding checks to the state transitions, if possible this is better than calling within code.

Reply:
I look into that, but as far as I know Animator parameters have to be set in code as well and cannot be inferred from existing variables (see line 91). In this case, I thought either setting some boolean in the Animator or playing the animation directly didn't make much difference imo

Reply:
I see, in this case I would still not call Play and set up a trigger variable